### ISSUE TITLE
Meteor issues

### ISSUE THREAD

**Issue by CptWad** [d3athrow/vgstation13#7735]
Meteors can destroy the emergency shielding, causing the emergency shield generators to bug out and not be able to reproduce shielding. 

Meteors will still fuck over everything and everyone. An extra 5 minutes of setup time might help but it seems nothing can stop meteors from killing all of the station

Flashbang meteors stun borgs, does not blind them but sure as hell makes them immoble

Players can escape the z-level to not get meteored. 

on turfs that cant be destroyed (The shuttle) A shitton of "Soot" is generated and stacks on said tiles after repeated meteoring

**Comment by DrCelt** [comment]
@dylanstrategie

**Comment by CptWad** [comment]
I would like to add to the fact that the emergency shielding, or any shielding does not protect from meteors. At all. Meaning you cant prepare yourself for the meteor waves at all.

**Comment by dylanstrategie** [comment]
First one shouldn't happen, meteors, and for that matter any explosion, should drain power from all hit shields, meaning too many hits at once on an unstable powergrid will down the shield

I think as far as "meteors destroying everything and everyone" goes, the main problem is that meteors were made overpowered in my last PR. That leaves me some room to tweak some values, I think inter-wave length is more important, then build time

Third one is pretty much intended, they should flick the flash overlay though, might be because it's cast to carbons

Fourth one is hard to fix properly. If I fix meteors as I intend, the meteors that miss the station will go fuck up over Z-levels

Fifth is a simple problem, need to only create soot if there's no soot on the same tile, I'll fix it asap

**Comment by gbasood** [comment]
Should test the first issue anyways, seems that other people have said something similar about shields

**Comment by CptWad** [comment]
yeah I had set up 4 emergency sheilding in the mommi nest and the first meteor that hit the sheild straight fucked everything. The blast ignored the fact that there was a sheild there and simply blew me and the other mommi up.

Really the only way to survive meteors is the know the few ways around it

>Teleporting off zlevel
>Getting in one of the two fridges
>laying down so the meteors fly over you
>A fourth one I have yet to have the opportunity to test

**Comment by gbasood** [comment]
First issue definitely happened. Watched a meteor destroy part of the shielding until it regenerated about 10 seconds later.

**Comment by AlphaBuffalo** [comment]
I set the shield recharge rate to 1 renwark/sec or what ever fake unit that game uses, and the strength ot 1 renwark, so in theory it should be regenerating a new shield black every second

**Comment by AlphaBuffalo** [comment]
What if we add more settings for the shield generators, like adding different shape configurations, increase recharge rate, and increase power usage

**Comment by Shadowmech88** [comment]
The soot, at least, appears to no longer happen. I haven't tested the other issues.
